src/app/component/user/components/profile/edit-profile/edit-profile.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="container">
+    <div class="title">
+        <h2>Edit profile</h2>
+    </div>
+    <app-personal-photo></app-personal-photo>
+    <form>
+        <div class="wrapper">
+            <p>Personal info</p>
+            <div class="input-block" [ngStyle]="{'margin-bottom': '41px'}">
```

Comment:
General: please, if you have more than 1 attribute, write them in a separate rows, for better code readability.

Reply:
fixed